source/features/limit-commit-title-length.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import onetime from 'onetime';
+import features from '../libs/features';
+
+const getError = onetime<[], HTMLElement>((): HTMLElement =>
+	<p className="note text-red">Title of commit should be less than 70 characters.</p>
+);
+
+function init(): void {
+	const inputs = select.all<HTMLInputElement>([
+		'#commit-summary-input', // Commit title on edit file page
+		'#pull_request_title', // PR title while creating PR
```

Comment:
This is unrelated. If the user wants the PR title to match the commit they’ll do so when merging the PR 🙂

Reply:
When creating a PR from a branch with a very long commit title, GitHub already separates out pull request title to 70 characters in title field and the rest into the message field.

So this should be a good practice for both the contributor (while opening PR) and for the owner (while merging PR) right!?